tsdb/db.go

```diff
@@ -510,7 +511,51 @@ func (db *DBReadOnly) loadDataAsQueryable(maxt int64) (storage.SampleAndChunkQue
 			}
 		}
 		opts := DefaultHeadOptions()
-		opts.ChunkDirRoot = db.dir
+		// opts.ChunkDirRoot = db.dir
+
+		// Just a PoC!!
+		// The idea here is to prevent the read head from altering RW head's MmappedChunksDir
```

Comment:
We could think about a mocked `ChunkDiskMapper` or sth, but that will complicate things + we may apply keeping more data on memory...